src/psyclone/tests/lfric_build.py

```diff
@@ -160,8 +161,8 @@ def _build_infrastructure(self):
         except (CompileError, IOError) as err:
             # Failed to compile one of the files
             LFRicBuild._infrastructure_built = False
-            raise CompileError("Could not compile LFRic wrapper. "
-                               "Error: {0}".format(str(err.value)))
+            raise CompileError(f"Could not compile LFRic wrapper. "
+                               f"Error: {str(err)}") from err
```

Comment:
Probably don't need the `str()` around `err` here.

Reply:
Removed.